extensions/amp-imgur/0.1/amp-imgur.js

```diff
@@ -79,10 +79,10 @@ export class AmpImgur extends AMP.BaseElement {
     iframe.setAttribute('frameborder', '0');
     iframe.setAttribute('allowfullscreen', 'true');
 
-    iframe.src =
-      'https://imgur.com/' +
-      encodeURIComponent(this.imgurid_) +
-      '/embed?pub=true';
+    const sanitizedID = startsWith(this.imgurid_, 'a/')
+      ? 'a/' + encodeURIComponent(this.imgurid_.replace('a/', ''))
```

Comment:
Would rewrite this as:

```javascript
const sanitizedId = this.imgurid_.replace(/^(a\/)?(.*)/, (match, part1, part2) => {
  return (part1 || '') + encodeURIComponent(part2);
});
```

Conceptually, you worry less about branching and the flow of the code and more on the parts of the data and what should get replaced.